namastack-outbox-tracing/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+    alias(libs.plugins.kotlin.spring)
+    jacoco
+}
+
+description = "namastack-outbox-tracing"
+
+dependencies {
+    implementation(project(":namastack-outbox-api"))
+    implementation(project(":namastack-outbox-core"))
+    implementation(libs.spring.boot.autoconfigure)
+    implementation(libs.spring.aspects)
+    implementation(libs.spring.boot.micrometer.tracing.opentelemetry)
```

Comment:
The tracing library should be based on the `micrometer-tracing` library, not on `opentelemetry`. Spring Boot uses Micrometer as its base, allowing users to choose which tracer library they prefer: OpenTelemetry, or OpenZipkin Brave. By supporting `micrometer-tracing`, we will support both.

https://docs.spring.io/spring-boot/reference/actuator/tracing.html